methods.rb

```diff
@@ -0,0 +1,121 @@
+class Methods
+  def initialize
+    @person_array = []
+    @books = []
+    @rental_array = []
+  end
+
+  def create_person
+    puts ' Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_option = gets.chomp.to_i
+
+    case person_option
+    when 1
+      print 'Age: '
+      user_age = gets.chomp.to_i
+
+      print 'Name: '
+      user_name = gets.chomp
+
+      print 'Has parent permission? [Y/N]: '
+      user_permission = gets.chomp.to_s.upcase
+
+      case user_permission
+      when 'Y'
+        user_permission = true
+      when 'N'
+        user_permission = false
+      end
+      student = Student.new(user_age, user_name, parent_permission: user_permission)
+      @person_array << {
+        output: "[Student] Name: #{student.name}, ID: #{student.id}, Age: #{student.age}",
+        object: student
+      }
+
+      puts 'Person created successfully!'
+      puts "\n"
+    when 2
+      print 'Age: '
+      user_age = gets.chomp.to_i
+
+      print 'Name: '
+      user_name = gets.chomp
+
+      print 'Specialization: '
+      user_specialization = gets.chomp
+
+      teacher = Teacher.new(user_age, user_name, user_specialization)
+      @person_array << {
+        output: "[Teacher] Name: #{teacher.name}, ID: #{teacher.id}, Age: #{teacher.age}",
+        object: teacher
+      }
+
+      puts 'Person created successfully!'
+      puts "\n"
+    else
+      puts 'Person not created.'
+    end
+  end
```

Comment:
- [ ] Great job refactoring 👍🏽 Creating the `method.rb file`.

Reply:
@Whoistolu  thank you so much